tests/test_operator.py

```diff
@@ -1269,120 +1269,6 @@ def test_loose_kwargs(self):
 @skipif('device')
 class TestDeclarator(object):
 
-    def test_heap_1D(self):
```

Comment:
all these tests were obsolete, a massive PITA to maintain, and basically unnecessary. Hence, dropping